frontends/election-manager/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -49,96 +43,45 @@ export interface Props {
 
 export function WriteInsAdjudicationScreen({
   contest,
-  writeInSummaryEntries,
   onClose,
   adjudicateTranscription,
   updateAdjudication,
 }: Props): JSX.Element {
-  const contestWriteInCount = writeInSummaryEntries.reduce(
-    (acc, entry) => acc + entry.writeInCount,
-    0
-  );
-
-  const adjudicatedGroups: AdjudicationGroup[] = [];
-  const pendingAdjudications: PendingWriteInAdjudication[] = [];
-
-  for (const [adjudicatedValue, entries] of groupBy(
-    writeInSummaryEntries,
-    (entry) => entry.writeInAdjudication?.adjudicatedValue
-  )) {
-    if (adjudicatedValue) {
-      adjudicatedGroups.push({
-        adjudicatedValue,
-        writeInCount: collections.reduce(
-          entries,
-          (acc, entry) => acc + entry.writeInCount,
-          0
-        ),
-        writeInAdjudications: Array.from(entries, (entry) => ({
-          id: entry.writeInAdjudication?.id as Id,
-          adjudicatedValue: entry.writeInAdjudication
-            ?.adjudicatedValue as string,
-          transcribedValue: entry.writeInAdjudication
-            ?.transcribedValue as string,
-          writeInCount: entry.writeInCount,
-        })),
-      });
-    } else {
-      for (const entry of entries) {
-        if (entry.transcribedValue) {
-          pendingAdjudications.push({
-            transcribedValue: entry.transcribedValue,
-            writeInCount: entry.writeInCount,
-          });
-        }
-      }
-    }
-  }
-
-  const adjudicationQueuePhrase = `${format.count(
-    pendingAdjudications.length
-  )} ${pluralize('transcriptions', pendingAdjudications.length)} to adjudicate`;
+  const writeInAdjudicationTableQuery = useWriteInAdjudicationTableQuery({
+    contestId: contest.id,
+  });
+  const { isLoading } = writeInAdjudicationTableQuery;
```

Comment:
Should this generally be how we handling loading with react-query? I think I have not been doing this.

Reply:
I'm unsure. Mostly I've just cared about whether `data` is loaded or not, but in this case I wanted a Boolean value so I pulled `isLoading` since it was already there.